python/metricq/connection_watchdog.py

```diff
@@ -1,4 +1,4 @@
-from asyncio import CancelledError, Event, Task, TimeoutError, ensure_future, wait_for
+from asyncio import CancelledError, Event, Task, TimeoutError, create_task, wait_for
```

Comment:
Looks like only `loop.create_task()` is used below, so no need to import `asyncio.create_task`:

```suggestion
from asyncio import CancelledError, Event, Task, TimeoutError, wait_for
```